src/thunderbots/software/backend_output/backend.h

```diff
@@ -10,8 +10,8 @@ class Backend
      *
      * @param primitives the list of primitives to send
      */
-    virtual void sendPrimitives(
-        const std::vector<std::unique_ptr<Primitive>> &primitives) = 0;
+    virtual void sendPrimitives(const std::vector<std::unique_ptr<Primitive>>& primitives,
```

Comment:
(1/2) Hmm... sorry for the late comment, but the modification to this base class doesn't sit well with me. My interpretation of this function in the backend is a stateless function that just translates primitives into whatever medium is used to communicate with "robots", and adding Team here breaks that abstraction.
Considering creating a helper function in the actual grsim_backend implementation that takes in the necessary Team parameter, and updating it as part of the main loop of the backend.

Reply:
Team object removed from sendPrimitives(). Helper function updateBackendTeam() added.